### PR TITLE
libretroshare: new port in net

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port: https://github.com/RetroShare/libretroshare
Its dependency: https://github.com/Corvusoft/restbed

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
@harens This one should be good to merge.

File: devel/restbed/Portfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                    -DBUILD_SSL=OFF \
+                    -DBUILD_TESTS=OFF
+
+pre-destroot {
+    xinstall -d ${destroot}${prefix}/include/${name}
+}
+
+post-destroot {
+    # Fix permissions; setting to 0644 does not have a needed effect.
+    file attributes ${destroot}${prefix}/include/${name} -permissions 0755
```
Comment (harens) [inline devel/restbed/Portfile:40]: Maybe rewrite the pre-destroot to the following:

```tcl
xinstall -d -m 0755 ${destroot}${prefix}/include/${name}
```

Comment (barracuda156) [reply devel/restbed/Portfile:40]: Should probably work, I can try.

Comment (barracuda156) [reply devel/restbed/Portfile:40]: @harens No, it does not. Permissions are set incorrectly then, and the directory is inaccessible. I do not know why, but guess just let it be how I have set it. I tried many options, this is what is confirmed to work.

**Comment by barracuda156** [comment]
@harens Can we merge this now?

**Pull request merged**